### ISSUE TITLE
Is this Libarary unusable now?

### ISSUE THREAD

**Issue by umang350** [jlobos/instagram-web-api#296]
It seems it doesn't works in any way, thus please add note to the readme for saving time of developers stumbling upon this library.
#291

**Comment by emauriciomoreno** [comment]
Yes, that's right, the library has stopped working. At the point where it gets the csrf_token, before the login process

**Comment by ubaidimranofficial** [comment]
I have a solution for that :_

**Comment by emauriciomoreno** [comment]
Please tell me

**Comment by ubaidimranofficial** [comment]
You can update this code, I just debugged the issue and solved it on a production level.  :)
Thanks, me later.
```
let value = this.request('/', { resolveWithFullResponse: true }).then(res => {
      const pattern = new RegExp(/(csrf_token\\":\\")[\w]+/)
      const matches = res.body.match(pattern)
      value = matches[0].substring(15)
    })
```

**Comment by emauriciomoreno** [comment]
Thanks a lot. you saved my day

**Comment by tusharsinghal1610** [comment]
Thanks a lot @ubaidimranofficial

**Comment by espressostack** [comment]
@ubaidimranofficial Thanks a lot bro

**Comment by DreamBoy65** [comment]
> You can update this code, I just debugged the issue and solved it on a production level. :) Thanks, me later.
> 
> ```
> let value = this.request('/', { resolveWithFullResponse: true }).then(res => {
>       const pattern = new RegExp(/(csrf_token\\":\\")[\w]+/)
>       const matches = res.body.match(pattern)
>       value = matches[0].substring(15)
>     })
> ```

now its giving:
![Screenshot_20221124-183505_Termux](https://user-images.githubusercontent.com/77263344/203791519-7fea51bb-adb7-4110-a423-4f8d79630ef7.jpg)

**Comment by quaint-racoon** [comment]
Got same error as the guy above any solutions?

**Comment by marcelo-earth** [comment]
> You can update this code, I just debugged the issue and solved it on a production level. :) Thanks, me later.
> 
> ```
> let value = this.request('/', { resolveWithFullResponse: true }).then(res => {
>       const pattern = new RegExp(/(csrf_token\\":\\")[\w]+/)
>       const matches = res.body.match(pattern)
>       value = matches[0].substring(15)
>     })
> ```

Wait, but where I put this?

**Comment by quaint-racoon** [comment]
> > You can update this code, I just debugged the issue and solved it on a production level. :) Thanks, me later.
> > 
> > ```
> > let value = this.request('/', { resolveWithFullResponse: true }).then(res => {
> >       const pattern = new RegExp(/(csrf_token\\":\\")[\w]+/)
> >       const matches = res.body.match(pattern)
> >       value = matches[0].substring(15)
> >     })
> > ```
> 
> Wait, but where I put this?




U edit the package code itself and replace the bugged code

**Comment by marcelo-earth** [comment]
> > > You can update this code, I just debugged the issue and solved it on a production level. :) Thanks, me later.
> > > ```
> > > let value = this.request('/', { resolveWithFullResponse: true }).then(res => {
> > >       const pattern = new RegExp(/(csrf_token\\":\\")[\w]+/)
> > >       const matches = res.body.match(pattern)
> > >       value = matches[0].substring(15)
> > >     })
> > > ```
> > 
> > 
> > Wait, but where I put this?
> 
> U edit the package code itself and replace the bugged code

Wow, okay. Thank you.

**Comment by ardinusawan** [comment]
For everyone who looking for solution:
Based on this youtube desc: https://www.youtube.com/watch?v=XzyYi_yv86A

1. Create an empty directory in the root of your app called "instagram-web-api"
2. Copy the index.js file from my GitHub repo ([https://github.com/amamenko/inky-dood...](https://github.com/amamenko/inky-doodle/blob/master/instagram-web-api/index.js) into that directory.
3. Substitute your current import of the "Instagram" object from the instagram-web-api library in your app with const Instagram = require("./instagram-web-api/index");
